data/gegno/gegno.txt

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2021 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+phrase "gegno vi"
+	word
+		""
+		""
+		Beuci
+		Dueyu
+		Tie
+		Efta
+		Gye
+		Kyhe
+		Efe
+		Eys
+		Imnn
+		Yx
+		Isyp
+		Tyyn
+		Hy
+		Eysyp
+		Iidt
+		Ych
+	word
+		" "
+	word
+		Oorevyt
+		Ufftye
+		IiL
+		Tyebbui
+		Yuduue
+		Uhmtyeph
+		Qesscit
+		Cscayim
+		Eisc
+		Iilsc
+		Eitch
+		Ooayvyt
+		Echtyya
+		Tyyasca
+		Iefsct
+		Ieftye
+		Iusta
+		Uffduyeu
+		Bbutye
+		Scbiye
+		Aehietch
+		Dueyoh
+		Uuftei
+		Ootyff
+
+phrase "gegno scin"
+	word
+		""
+		""
+		"Kuuay"
+		Jiee
+		Tyuoe
+		Aur
+		Yss
+		Zueyt
+		Zo
+		Tschzer
+		Lleiesid
+		Inyseiet
+		Dy
+		Ispue
+		Jeiys
+		Iepy
+	word
+		" "
+	word
+		Oorevyt
+		IiL
+		Iievae
+		Gyteienl
+		Piaxenni
+		Ufftye
+		Echtyya
+		Duyeitch
+		Ziurteil
+		Ymiffeft
+		Tchynd
+		Tscael
+		Ymipiae
+		Eisc
+		Pixeni
+		Tiyesca
+		Vduscae
+		Duefyu
+		Llteiurv
+		Enncyurtte
+
+phrase "gegno uncontacted"
+	word
+		"(A deep disgruntled voice says something in a language you cannot understand.)"
+		"(A low ominous voice speaks in a manner you have not heard before.)"
```

Comment:
The reason I went for two different lines here is for more flavor, and the Gegno _are_ split up into two factions, so I don't see why having two different hail phrases wouldn't be fitting. (Although Derpy and others may suggest otherwise.)

Reply:
My logic for this suggestion is that `manner` suggests maybe a different accent, or a different tone of voice. To me, it doesn't imply that they're speaking in a entirely different language. However, it's not really a big deal, you can ignore this suggestion if you want.